packages/xy-chart/src/selection/Brush.jsx

```diff
@@ -109,10 +118,16 @@ class Brush extends React.Component {
     return domain;
   }
 
-  handleBrushStart() {
-    const { onBrushStart } = this.props;
+  handleBrushStart(point) {
+    const { x, y } = point;
+    const { onBrushStart, xScale, yScale } = this.props;
+    const invertedX = xScale.invert(x);
```

Comment:
does this work with band scales? 

possible solution [here](https://stackoverflow.com/a/30743306/1933266) if not (or we could punt to future PR / create issue).

Reply:
good catch! Indeed it doesnt work for band scales.. will revise it in this PR.